browser/p3a/p3a_core_metrics.h

```diff
@@ -63,6 +64,29 @@ class BraveWindowTracker : public BrowserListObserver {
   DISALLOW_COPY_AND_ASSIGN(BraveWindowTracker);
 };
 
+class BraveNewTabCountTracker
+     : public ::ntp_background_images::ViewCounterService::Observer {
+ public:
+  explicit BraveNewTabCountTracker(PrefService* local_state);
+  ~BraveNewTabCountTracker() override;
+
+  static void CreateInstance(PrefService* local_state);
+
+  static void RegisterPrefs(PrefRegistrySimple* registry);
+
+  // ViewCounterService::Observer:
+  void OnBrandedWallpaperShown() override;
+  void OnNewTabOpened() override;
+
+ private:
+  void UpdateP3AValues() const;
+
+  WeeklyStorage new_tab_count_state_;
+  WeeklyStorage branded_new_tab_count_state_;
+
+  DISALLOW_COPY_AND_ASSIGN(BraveNewTabCountTracker);
```

Comment:
this macro is deprecated in favour of explicitly deleting copy c-tor and assignment operator